resources/graphql/authorization.graphql

```diff
@@ -48,20 +52,33 @@ type Tokens {
   message: String
 }
 
+type ApprovedStudy {
+  studyName: String
+  studyAbbreviation: String
+}
 
 input UpdateUserInput {
     firstName: String!
     lastName: String!
 }
 
+input ApprovedStudyInput {
+  studyName: String!
+  studyAbbreviation: String!
+}
+
 type Query {
     "Limit to current user only"
     getMyUser : User
-    listOrganizations: [OrgInfo]
 
     "Admin and Org Owner only operations, Org Owner only see users within same organization"
     listUsers: [User]
+    listActiveCurators: [UserInfo]
     getUser(userID: ID!): User
+
+    "Admin only operations"
+    listOrganizations: [Organization]
```

Comment:
I think it makes sense to change the return type of `listOrganizations` from `OrgInfo` to `Organization`, but we will need to change both the backend and frontend implementation for this endpoint. This change will break the current implementation for user management.